docs/1.5/wcc-contentful-middleman/WCC/Contentful/Middleman/Extension.html

```diff
@@ -0,0 +1,347 @@
+
+  <span class='comment'># set up your extension
+</span>  <span class='kw'>return</span> <span class='kw'>if</span> <span class='const'><span class='object_link'><a href="../../../WCC.html" title="WCC (module)">WCC</a></span></span><span class='op'>::</span><span class='const'><span class='object_link'><a href="../../Contentful.html" title="WCC::Contentful (module)">Contentful</a></span></span><span class='period'>.</span><span class='id identifier rubyid_initialized'>initialized</span>
+
+  <span class='const'><span class='object_link'><a href="../../../WCC.html" title="WCC (module)">WCC</a></span></span><span class='op'>::</span><span class='const'><span class='object_link'><a href="../../Contentful.html" title="WCC::Contentful (module)">Contentful</a></span></span><span class='period'>.</span><span class='id identifier rubyid_configure'>configure</span> <span class='kw'>do</span> <span class='op'>|</span><span class='id identifier rubyid_config'>config</span><span class='op'>|</span>
+    <span class='id identifier rubyid_config'>config</span><span class='period'>.</span><span class='id identifier rubyid_store'>store</span> <span class='symbol'>:eager_sync</span><span class='comma'>,</span> <span class='symbol'>:memory</span>
+
+    <span class='id identifier rubyid_options'>options</span><span class='period'>.</span><span class='id identifier rubyid_to_h'>to_h</span><span class='period'>.</span><span class='id identifier rubyid_each'>each</span> <span class='kw'>do</span> <span class='op'>|</span><span class='lparen'>(</span><span class='id identifier rubyid_k'>k</span><span class='comma'>,</span> <span class='id identifier rubyid_v'>v</span><span class='rparen'>)</span><span class='op'>|</span>
+      <span class='id identifier rubyid_config'>config</span><span class='period'>.</span><span class='id identifier rubyid_public_send'>public_send</span><span class='lparen'>(</span><span class='tstring'><span class='tstring_beg'>&quot;</span><span class='embexpr_beg'>#{</span><span class='id identifier rubyid_k'>k</span><span class='embexpr_end'>}</span><span class='tstring_content'>=</span><span class='tstring_end'>&quot;</span></span><span class='comma'>,</span> <span class='id identifier rubyid_v'>v</span><span class='rparen'>)</span> <span class='kw'>if</span> <span class='id identifier rubyid_config'>config</span><span class='period'>.</span><span class='id identifier rubyid_respond_to?'>respond_to?</span><span class='lparen'>(</span><span class='tstring'><span class='tstring_beg'>&quot;</span><span class='embexpr_beg'>#{</span><span class='id identifier rubyid_k'>k</span><span class='embexpr_end'>}</span><span class='tstring_content'>=</span><span class='tstring_end'>&quot;</span></span><span class='rparen'>)</span>
+    <span class='kw'>end</span>
+
+    <span class='comment'># TODO: create MiddlemanRichTextRenderer and set as default here
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/watermarkchurch/wcc-contentful/blob/4b7ad73977bb23e3609efacfef4a6001e768e62a/docs/1.5/wcc-contentful-middleman/WCC/Contentful/Middleman/Extension.html#L262"></span>TODO added in <a href="https://github.com/watermarkchurch/wcc-contentful/blob/4b7ad73977bb23e3609efacfef4a6001e768e62a/docs/1.5/wcc-contentful-middleman/WCC/Contentful/Middleman/Extension.html">docs/1.5/wcc-contentful-middleman/WCC/Contentful/Middleman/Extension.html</a> - is there a card associated with that?</td>
    </tr>
  </tbody>
</table>